### ISSUE TITLE
Add speed to StumblerBundle

### ISSUE THREAD

**Issue by crankycoder** [mozilla/MozStumbler#1497]
While working on porting MozStumbler to use geolocate and geosubmit, I've noticed that we can add speed to the stumbler bundle.

Grab it from the Location object and check the hasSpeed() method first.

https://github.com/crankycoder/MozStumbler/blob/8ae4bbecaec883f06897e76d0a89de9b73457609/libraries/stumbler/src/main/java/org/mozilla/mozstumbler/service/stumblerthread/datahandling/StumblerBundle.java#L117

**Comment by hannosch** [comment]
Yes, we have both a new `speed` and `heading` field in the API. Those actually got added to the submit API as well, put where never provided by mozstumbler.

**Comment by hannosch** [comment]
Other differences between the submit / geosubmit API's (not to mention the lower vs. camel case):
- A new `timestamp` field in ms since the unix epoch, no more `time` field using an ISO date string.
- Always put the `radioType` field inside each cell record, don't provide a `radioType` at the top report level - this removes the weirdness with phone type vs. cell type.
- Wifi records have a new `signalToNoiseRatio` field.
